gravitee-apim-gateway/pom.xml

```diff
@@ -122,6 +122,12 @@
             <artifactId>junit-jupiter-params</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.pivovarit</groupId>
+            <artifactId>throwing-function</artifactId>
+            <version>${throwing-function.version}</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
No sure that this import is used :thinking:

Reply:
woops I forgot to remove it.